projects/jdwp/defs/schema.py

```diff
@@ -46,9 +69,11 @@ class TaggedUnion(Enum):
 
 Type = PrimitiveType | Array | TaggedUnion
 
+T = TypeVar("T")
+
 
 @dataclass(frozen=True)
-class Field:
+class Field(Generic[T]):
     """Field class."""
 
     name: str
```

Comment:
Could you change the line below to be `type: T` ? We want `Field[ArrayLength]` to be parametrized with a value of type `ArrayLength`